source/docs/changelog.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
```

Comment:
@octoper this is not the good format tho. Can you double check?

Reply:
Just wondering, how do you want it displayed? 🙂 Guess it needs to be added to [the nav](https://github.com/pestphp/website/blob/master/navigation.php#L38) when it's ready.

Reply:
@nunomaduro Yeah you are right! I'm dump, I will fix it

Reply:
We could go with Tailwind's docs idea of having more separate sections:

```
Getting Started:

- Installation
- Release Notes


Using Pest:

- Writing Tests
- Underlying Test Case
- …
```

<img width="252" alt="Capture d’écran 2020-06-05 à 10 56 59" src="https://user-images.githubusercontent.com/19224681/83857473-5463be80-a71b-11ea-9433-7075967b0807.png">